test/Microsoft.Azure.WebJobs.Host.EndToEndTests/AzureStorageEndToEndTests.cs

```diff
@@ -191,13 +191,13 @@ public AzureStorageEndToEndTests(TestFixture fixture)
             await EndToEndTest(uploadBlobBeforeHostStart: false);
         }
 
-        [Fact]
+        [Fact(Skip = "This test relies on ICloudBlobStreamBinder<T> support, whith has been removed")]
```

Comment:
Fix spelling error, and reference the bug https://github.com/Azure/azure-webjobs-sdk/issues/995

Reply:
done.